helix-term/src/commands.rs

```diff
@@ -4564,6 +4565,11 @@ fn insert_register(cx: &mut Context) {
     })
 }
 
+fn clear_registers(cx: &mut Context) {
+    cx.editor.registers.clear();
+    cx.editor.set_status("All registers cleared.")
+}
```

Comment:
If we have the typable command `:clear-register` then we don't need the regular command here. You can create keybindings that use `:clear-register`.

Reply:
I see this one is clearing all registers while the typable command clears the given register. The typable command could clear all registers if no arguments are provided.

(Side-note: this would be a nice application for typable command flags, like `:clear-register --all`. I'll make an issue for that)

Reply:
Good point. I left only the typable `:clear-register`